open-source.adoc

```diff
@@ -384,37 +382,37 @@ and would feel rude going back to "does this make sense" after the
 author has addressed 20+ of my code organization suggestions.
 ____
 
-Her view, that a good review should focus on what's most needed at
-this point in time, aligns well with Jon Atack's writings. She
-proposes lots of questions at various levels of review, but stresses
-that this list is not in any way exhaustive or that it should be used
-as a straight-out recipe. The list is illustrated with real-life
+Her idea that a good review should focus on what's most needed at
+a specific point in time aligns well with Jon Atack's advice. She
+proposes a list of questions that you may ask yourself at various levels of the review process, but stresses
+that this list is not in any way exhaustive nor
+a straight-out recipe. The list is illustrated with real-life
 examples from GitHub.
 
 === Funding
 
-Lots of people work with Bitcoin open source development, for Bitcoin
-Core or for other projects. Many do it on their spare time without
-getting any compensation, but some developers are also getting paid to
+Lots of people work with Bitcoin open source development, either for Bitcoin
+Core or for other projects. Some do it in their spare time without
+getting any compensation, but many developers are also getting paid to
```

Comment:
Haha, I recently changed this from "many" to "some" and you now change it back. Most people do it for free in their spare time and a few get paid. I think the sentence as it's written now gives the wrong impression.

Reply:
You're right, it seems to suggest that there are more paid than unpaid devs. I'll change it back

Reply:
Also change back the first word "Some" to "Many". "Many do it in their spare time ..."